libqtile/widget/tasklist.py

```diff
@@ -231,11 +238,14 @@ def get_taskname(self, window):
         elif window.maximized:
             state = self.txt_maximized
             markup_str = self.markup_maximized
+        elif window is window.group.current_window:
+            if window.floating:
+                markup_str = self.markup_focused_floating
```

Comment:
```suggestion
                markup_str = self.markup_focused_floating or self.markup_floating
```
I think your code introduces a change for current users as a focused floating window would no longer use `markup_floating`. So, if users don't want to distinguish floating windows, we should fall back to `markup_floating`.

Reply:
> if users don't want to distinguish floating windows, we should fall back to `markup_floating`

Right, I didn't think about that.
For testing what should I do? There is no test file for this widget

Reply:
Oh. Maybe I need to write a basic test first and then you can add your extra bits after that.

Reply:
Once #4218 is merged then you can add your own test to that file.

Reply:
Yes, but only once that PR has been merged.